tests/core.py

```diff
@@ -1163,6 +1163,22 @@ def test_fetch_task_instance(self):
         response = self.app.get(url)
         assert "runme_0" in response.data.decode('utf-8')
 
+    def test_task_info(self):
```

Comment:
within the test folder (I think there are some examples). The idea is to have the test folder reflect the main structure, then tests are easier to find.